### PR TITLE
doc: release notes: 2.0 Kernel release notes

### PR DESCRIPTION
**Pull request by carlescufi**
Add a section for the kernel release notes for the 2.0 release.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

File: doc/releases/release-notes-2.0.rst [hunk 1]
```diff
@@ -28,6 +28,13 @@ Kernel
 
 * New kernel API for per-thread disabling of Floating Point Services for
   ARC, ARM Cortex-M, and x86 architectures.
+* New system call to set the clock frequency at runtime.:w
```
Comment (ioannisg) [inline doc/releases/release-notes-2.0.rst]: what is :w?

**Review by ioannisg — APPROVED** [review]
A minor comment, otherwise looks good

File: doc/releases/release-notes-2.0.rst (hunk 1 above)
Comment (carlescufi) [reply doc/releases/release-notes-2.0.rst]: It's the save function in `vim` :D

Comment (carlescufi) [reply doc/releases/release-notes-2.0.rst]: fixed

**Review by dbkinder — APPROVED** [review]

**Comment by dbkinder** [comment]
@ioannisg Can you merge this so I can make a pass over all the release notes and not cause a merge conflict.

**Pull request merged**